pkg/reconciler/mtnamespace/namespace.go

```diff
@@ -120,11 +117,11 @@ func (r *Reconciler) reconcileBroker(ctx context.Context, ns *corev1.Namespace)
 	// If the resource doesn't exist, we'll create it.
 	if k8serrors.IsNotFound(err) {
 		b := resources.MakeBroker(ns)
-		b, err = r.EventingClientSet.EventingV1beta1().Brokers(ns.Name).Create(b)
+		b, err = r.eventingClientSet.EventingV1beta1().Brokers(ns.Name).Create(b)
 		if err != nil {
 			return nil, err
 		}
-		r.Recorder.Event(ns, corev1.EventTypeNormal, brokerCreated,
+		r.recorder.Event(ns, corev1.EventTypeNormal, brokerCreated,
```

Comment:
as above, these should be using `controller.GetEventRecorder(ctx)`

Reply:
I am doing the thinest change

Reply:
this is a k8s type `func (r *Reconciler) Reconcile(ctx context.Context, key string) error {`